src/admin-views/widgets/components/radio.php

```diff
@@ -29,15 +29,15 @@ class="tribe-widget-form-control tribe-widget-form-control--radio"
 >
 	<input
 		class="tribe-widget-form-control__input"
-		id="<?php echo esc_attr( $id ); ?>"
+		id="<?php echo esc_attr( $id ) . '-' . strtolower( esc_attr( $button_value ) ); ?>"
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'strtolower'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: d868937ea6dceb2e75d0e1c7f682efc1</sub>